assets/styles/tools/_functions.scss

```
@@ -139,3 +139,50 @@
 }
 
 $context: 'frontend' !default;
+
+
+
+/**
+ * Calculate grid space
+ * @param  {number} $number The percentage spacer
+ * @return {number} in calc
+ */
+@function grid-space($percentage, $offset: 0) {
```

Improve documentation and use Sass comment syntax since this is a Sass function.

```suggestion
// Returns calculation of a percentage of the grid cell width
// with optional offset of grid gutter.
//
// ```scss
// .box {
//     width: grid-space(6/12);
//     margin-left: grid-space(1/12, 1);
// }
// ```
//
// @param  {number} $percentage
// @param  {number} $offset
// @return {function<number>}

@function grid-space($percentage, $offset: 0) {
```